src/arch/riscv/config.cmake

```diff
@@ -48,6 +50,12 @@ if (KernelSel4ArchRiscV32)
     set(KernelPTLevels 2 CACHE STRING "" FORCE)
 endif()
 
+config_option(KernelBuildRocketChip BUILD_ROCKET_CHIP "Build application to \
```

Comment:
[ditto](https://github.com/seL4/seL4/pull/94/files#r204326181), please rename this config to your specific board

Reply:
will rename BUILD_ROCKET_CHIP to BUILD_ROCKET_CHIP_ZEDBOARD